typescript/web/src/components/website/Features/Features.tsx

```diff
@@ -46,33 +48,55 @@ export const Features = () => {
               algorithms has never been that easy
             </Text>
           </Box>
+          <Img
+            htmlWidth="500px"
+            htmlHeight="320px"
+            height={{ md: "320px" }}
+            objectFit="contain"
+            src="/static/img/feature-picture.jpg"
+            alt="Electrical grid picture"
+          />
         </SimpleGrid>
         <SimpleGrid columns={{ base: 1, md: 3 }} mt="16" spacing="8">
-          <Feature
-            icon={BsFillLightningFill}
-            title="Blazing Fast Image Labeling Tool"
-          >
+          <Feature icon={BsFillLightningFill} title="Build datasets at scale">
             LabelFlow is an image labeling tool designed for optimum
             productivity. Keyboard shortcuts, interface layout, collaboration,
             everything is designed to build the most accurate datasets for
             machine learning.
           </Feature>
-          <Feature
-            icon={BsFillShieldLockFill}
-            title="Own your data and algorithms"
-          >
-            LabelFlow image annotation tool does not try to own your data or
-            algorithms, but integrates with them seamlessly. No duplicate source
-            of truth and complicated scripts to synchronize your data between
-            various tools.
+          <Feature icon={BsFillEmojiSmileFill} title="Optimum user experience">
+            Our obsession is to develop and keep developing a simple product. We
+            prefer you to focus on building the next big thing with AI rather
+            than finding a button to upload images.
           </Feature>
           <Feature icon={BsHeartFill} title="Open community and standard">
-            LabelFlow is building a community around an open source data
-            labeling tool (under Business Source Licence) to set the standard
-            around visual data management. Data cleaning should not require any
-            &ldquo;secret sauce&rdquo;.
+            We believe that collaboration could be the new word for competition.
+            We have made our{" "}
+            <Link color="brand.600" href="https://labelflow.ai/open">
+              KPIs public
+            </Link>
+            , our code is open under Business Source Licence and that is just
+            the beginning.
           </Feature>
         </SimpleGrid>
+        <Stack
+          direction={{ base: "column", md: "row" }}
+          justifyContent="space-around"
+          mt="8"
+        >
+          <NextLink href="/auth/signin">
+            <Button
+              size="lg"
+              minW="210px"
+              colorScheme="brand"
+              height="14"
+              px="8"
+              rightIcon={<BsArrowRight />}
+            >
+              Try it now
+            </Button>
+          </NextLink>
```

Comment:
This is the same code as in `Hero.tsx`. We should factorize and use a common `TryItNowButton` in both places.